cos-gpu-installer-docker/entrypoint.sh

```diff
@@ -363,7 +382,7 @@ set_compilation_env() {
         # variable based on the toolchain used for kernel compilation
         source "${TOOLCHAIN_ENV_FILENAME}"
         # Downloading toolchain from COS GCS Bucket
-        TOOLCHAIN_DOWNLOAD_URL="${COS_DOWNLOAD_GCS}/${BUILD_ID}/${TOOLCHAIN_ARCHIVE}"
+        TOOLCHAIN_DOWNLOAD_URL=$(get_cross_toolchain_pkg)
```

Comment:
Why change this?

Reply:
container-vm-image-staging bucket doesn't have toolchain tarball present in gcs bucket. To have a simpler code and interface, I opted for using `get_cross_toolchain_pkg`.
Calling this common API for both cos-cloud and container-vm-image-staging will lead to first checking in `CROS_GCS_BUCKET` and then fallback to `COS_CLOUD`.